client/src/components/GroupPage/GroupPage.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import './GroupPage.css';
+import GroupCard from "../GroupCard/GroupCard";
+import Navbar from '../Navbar/Navbar';
+import Footer from '../Footer/Footer';
+import card1 from "assets/images/card-image-3.png";
+import card2 from "assets/images/card-image-2.png";
+import card3 from "assets/images/card-image.png";
+import people from "assets/images/profile_default.svg";
+
+
+const GroupPage = props => {
+   
+    return (
+        <div>
+            <Navbar 
+            to={"/member-page"} 
+            profileImage={people}
+            />
+            
+            
+            <div className='wrapper'>
+                <h1>Groups You Can Join!</h1>
+            </div>
+            <div className="HomePage__group-cards-container">
+            <span className="HomePage__groupcard-1">
```

Comment:
You should never have a number in a class - by definition, classes should apply to multiple elements (as opposed to IDs)

Also, make sure your names are consistent (i.e. `HomePage__group-cards` v. `HomePage__groupcard`)